### PR TITLE
[Commerce] Add popup status for new cart addition

### PR DESCRIPTION
**Pull request by raisaveuc**

### CODE REVIEW

File: css/templates/e-commerce/page.css [hunk 1]
```diff
@@ -176,17 +176,17 @@ dd:after {
 }
 
 .ampstart-btn {
+  border-color: var(--color-primary);
   display: inline-block;
+  font-size: var(--button-font-size);
+  font-family: var(--title-font);
+  font-weight: var(--bold-font-weight);
+  padding: var(--space-1) var(--space-4);
```
Comment (raisaveuc) [inline css/templates/e-commerce/page.css]: Moved from below, styles that are shared between primary and secondary buttons

File: css/templates/e-commerce/page.css [hunk 2]
```diff
@@ -603,6 +603,52 @@ amp-selector [option][selected] {
   }
 }
 
+.commerce-cart-notification .commerce-cart-icon,
+.commerce-checkout .commerce-cart-icon {
+  display: none;
```
Comment (raisaveuc) [inline css/templates/e-commerce/page.css]: Moved from line 702 to combine shared styles

File: templates/e-commerce/cart.amp.json [hunk 3]
```diff
@@ -8,6 +8,11 @@
       ]
     }
   },
+  "cart": {
+    "item": {
+      "classes": "mx2 mb2 md-mx4 md-mt3 md-pb3"
```
Comment (raisaveuc) [inline templates/e-commerce/cart.amp.json]: Classes moved here for custom margins/padding across 3 pages

File: css/templates/e-commerce/page.css [hunk 4]
```diff
@@ -603,6 +603,52 @@ amp-selector [option][selected] {
...
+    opacity: 0;
+  }
+  15%, 85% {
+    opacity: 1;
+  }
+}
+
+@media (--breakpoint-md) {
+  .commerce-cart-notification {
+    top: 4rem;
```
Comment (gurmukhp) [inline css/templates/e-commerce/page.css]: Change to css var for navbar height wide.

File: templates/e-commerce/data.json [hunk 5]
```diff
@@ -157,7 +157,7 @@
         "description": "Mauris tincidunt justo convallis.",
         "price": "349",
         "quantity": 1,
-        "thumbnail": "../../../img/e-commerce/product/product-9.jpg"
+        "thumbnail": "../../../img/e-commerce/product/product-1.jpg"
```
Comment (gurmukhp) [inline templates/e-commerce/data.json]: Can we make sure the prices, product names also match?

**Comment by raisaveuc** [comment]
Updated with merge conflict resolution

**Pull request merged**